docs/package.json

```diff
@@ -55,7 +55,7 @@
   "type": "module",
   "dependencies": {
     "@web3-onboard/coinbase": "^2.1.4",
-    "@web3-onboard/core": "^2.11.1-alpha.1",
+    "@web3-onboard/core": "^2.12.0",
     "@web3-onboard/dcent": "^2.2.2",
     "@web3-onboard/enkrypt": "^2.0.0",
     "@web3-onboard/fortmatic": "^2.0.14",
```

Comment:
@taylorjdawson could you remove the alpha tags from the other web3-onboard packages while your in there?

Reply:
Thank you!